src/urllib3/util/ssl_.py

```diff
@@ -36,6 +36,19 @@ def _is_ge_openssl_v1_1_1(
     )
 
 
+def _is_ge_openssl_v1_1_1l(
+    openssl_version_text: str, openssl_version_number: int
+) -> bool:
+    """Returns True for OpenSSL 1.1.1l+ (>=0x101010cf)
+    LibreSSL reports a version number of 0x20000000 for
+    OpenSSL version number so we need to filter out LibreSSL.
+    """
+    return (
+        not openssl_version_text.startswith("LibreSSL")
+        and openssl_version_number >= 0x101010CF
```

Comment:
How do I know this? I currently have 1.1.1l on my laptop.

I considered using OPENSSL_VERSION_INFO which is less obscure and more fitting for Python code (which can compare tuples easily) but was lazy. Please tell me if you'd prefer that.